### DISCUSSION TITLE
Request - Auto-Stop at Chapter End Option

### DISCUSSION THREAD

**Discussion by ghost** [PaulWoitaschek/Voice#1307] [Ideas]
Hi! I would like to request for a feature, which is the option to auto-stop the playback at the end of each chapter--like I have a book that has separate files for each chapter (in a folder), and I would prefer to just play a chapter and then stop the playback automatically (sometimes I tend to forget because I ended up sleeping) Thanks for the help!

Please include the following details in your bug report:

App-Version: 3.6.3
Phone Name: Samsung Galaxy A7
Android Version: Lollipop

**Comment by mesonium** [comment]
Sounds like a duplicate of https://github.com/PaulWoitaschek/Voice/discussions/1323.
